server/lease/metrics.go

```diff
@@ -49,11 +49,19 @@ var (
 			// 1 second -> 3 months
 			Buckets: prometheus.ExponentialBuckets(1, 2, 24),
 		})
+
+	leaseTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "etcd_debugging",
+		Subsystem: "lease",
+		Name:      "total",
```

Comment:
```suggestion
		Name:      "active",
```
Looking at the naming scheme above we have "granted_total", "revoked_total", "renewed_total", so if in the Help you called them "active leases" then let's put it in the name of the metric.

In prometheus conventions https://prometheus.io/docs/practices/naming/, suffix total is reserved for accumulating metrics like counters, for gauges we should skip it.